tests/system.py

```diff
@@ -205,6 +220,7 @@ def test_should_be_able_to_get_results_from_query(self, gbq_connector):
         assert pages is not None
 
 
+@pytest.mark.local_auth
```

Comment:
It seems like `test_get_application_default_credentials_does_not_throw_error` and `test_get_application_default_credentials_returns_credentials` are working just fine on Travis if you want to move this to the `test_get_user_account_credentials_XXX` methods.